webapp/app.py

```diff
@@ -165,34 +168,52 @@ def global_template_context():
             component_tabs_file.read(), Loader=yaml.FullLoader
         )
 
+    updated_features = {}
+    for feature in FEATURES_LIST[0]["features"]:
+        feature_url = feature["url"].split("#")[0]
+        if feature_url not in updated_features:
+            updated_features[feature_url] = feature["status"]
+
     return {
         "version": VANILLA_VERSION,
         "versionMinor": version_minor,
         "path": flask.request.path,
         "page_tabs": component_tabs.get(docs_slug),
         "slug": docs_slug,
+        "releaseNotes": FEATURES_LIST,
+        "updatedFeatures": updated_features,
     }
 
+
 @app.template_filter()
 def markdown(text):
     return markupsafe.Markup(mistune.markdown(text))
 
+
 def class_reference(component=None):
-    component = component or urllib.parse.urlsplit(flask.request.path).path.split('/')[-1]
+    component = (
+        component
+        or urllib.parse.urlsplit(flask.request.path).path.split("/")[-1]
+    )
     data = CLASS_REFERENCES["class-references"][component]
-    return markupsafe.Markup(flask.render_template("_layouts/_class-reference.html", data=data))
+    return markupsafe.Markup(
+        flask.render_template("_layouts/_class-reference.html", data=data)
+    )
+
 
 @app.context_processor
 def utility_processor():
-    return {
-        "class_reference": class_reference,
-        "image": image_template
-    }
+    return {"class_reference": class_reference, "image": image_template}
 
 
 template_finder_view = TemplateFinder.as_view("template_finder")
 
 
+@app.route("/whats-new")
```

Comment:
Is this new route needed?

It seems it works fine as it is, under `/docs/what-new`